auth-web/src/components/auth/TransactionsDataTable.vue

```diff
@@ -134,35 +138,48 @@ export default class TransactionsDataTable extends Vue {
       description: 'Funds received'
     },
     {
-      status: TransactionStatus.CREATED,
-      description: 'Transaction Created'
+      status: TransactionStatus.PENDING,
+      description: 'Transaction is Pending'
     },
     {
-      status: TransactionStatus.DELETED,
-      description: 'Transaction Deleted'
+      status: TransactionStatus.CANCELLED,
+      description: 'Transaction is Cancelled'
     }
   ]
 
   private get getPaginationOptions () {
     let pagination = [...Array(this.PAGINATION_COUNTER_STEP)].map((value, index) => this.ITEMS_PER_PAGE * (index + 1))
```

Comment:
return directly if its not a problem.. just writing a comment on what the method expects to do will be helpful